spanner/api/Spanner.Samples/AddAndDropDatabaseRoleAsync.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_add_and_drop_database_role]

```

Comment:
Can you please revisit these with the Spanner team?

Best practices are to include only one sample per file/region tag: https://googlecloudplatform.github.io/samples-style-guide/#one-per-file

Reply:
We asked the same to the spanner team. They say its part of some documentation and because of which it is defined like this. 
ref: https://docs.google.com/document/d/17vh8nazkkdiZBJ8D7PjGFZuTR1Amwp3-F8o50AnuFMc/edit#

Reply:
I've left a comment on the doc pointing to the guidelines.